postcss.config.js

```diff
@@ -16,7 +16,7 @@
  */
 
 module.exports = {
-  plugins: {
-    autoprefixer: {}
-  }
+  plugins: [
+    require('autoprefixer')
```

Comment:
Yarn failed to build due to the old syntax? :man_shrugging: copied this from the autoprefixer docs.